tools/mason/MasonNew.chpl

```diff
@@ -114,12 +114,13 @@ private proc addGitIgnore(name: string) {
 
 
 proc makeBasicToml(name: string) {
-  const baseToml = '\n[brick]\n' +
+  const baseToml = '[brick]\n' +
```

Comment:
It might be cleaner to utilize a multi-line string with a format call here, e.g.

```chpl
const baseToml = '''[brick]
name = %s
version = "0.1.0"
...
""".format(name, ...);